crates/memory/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
```

Comment:
Memory module for testing and whatever else it's useful for

Reply:
In my [peer store wip](https://github.com/holochain/kitsune2/pull/19/files#diff-72a2ccbabb7487b60b153d720970a3bf4ce28548158ec52e4ac35a4a93b00e8dR1) I put the memory store in the `kitsune2` crate itself. I feel like these will be small enough an useful enough for testing, that maybe we don't need separate crates for them. Also the rust compiler is good at dead code elimination : )

Reply:
The in memory store you've created really is production ready and matches how the store should work in production. This one really isn't a production implementation because without persistence, why go to all this work to sync ops between peers. 

My thinking with this is that I need the in-memory implementations is crates that shouldn't depend on `kitsune2` because `kitsune2` needs to depend on them. So I think this does need to stay outside `kitsune2`

Reply:
That makes sense. And it's not too hard to re-arrange if we want to make a different choice in the future :+1:

Reply:
Agreed, we should take a good look at the crate layout before we publish to crates.io for the first time :)